x/sequencers/keeper/msg_server.go

```diff
@@ -0,0 +1,112 @@
+package keeper
+
+import (
+	"context"
+
+	errorsmod "cosmossdk.io/errors"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dymensionxyz/dymension-rdk/x/sequencers/types"
+	"github.com/dymensionxyz/gerr-cosmos/gerrc"
+)
+
+type msgServer struct {
+	Keeper
+}
+
+func NewMsgServerImpl(keeper Keeper) types.MsgServer {
+	return &msgServer{Keeper: keeper}
+}
+
+func (m msgServer) CreateSequencer(goCtx context.Context, msg *types.MsgCreateSequencer) (*types.MsgCreateSequencerResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	accAddr := msg.MustAccAddr() // ensured in validate basic
+	allow, err := m.IsSigned(ctx, accAddr, msg.GetKeyAndSig(), msg.GetPayload())
```

Comment:
why not `m.VerifySig` and remove the boolean arg

Reply:
ok, refactored a bit 👍